index.html

```diff
@@ -4,484 +4,528 @@
+                  />
                 </div>
-                <h2 class="job-block__title">Digital Marketing</h2>
+                <p class="job-block__title">Digital Marketing</p>
                 <p class="job-block__subtitle">150 Job Vacancy</p>
               </a>
 
-              <a href="#" class="content-container__job-blocks job-block">
+              <a href="#" class="jobs-group__job-blocks job-block">
                 <div class="job-block__circle">
                   <img
+                    class="job-block__img"
                     src="./src/icons/technical-support.svg"
                     alt="technical support"
                   />
+
+                  <img
+                    class="job-block__img_active"
+                    src="./src/icons/technical-support-active.svg"
+                    alt="technical support"
+                  />
                 </div>
-                <h2 class="job-block__title">Technical Support</h2>
+                <p class="job-block__title">Technical Support</p>
                 <p class="job-block__subtitle">350 Job Vacancy</p>
               </a>
 
-              <a href="#" class="content-container__job-blocks job-block">
+              <a href="#" class="jobs-group__job-blocks job-block">
                 <div class="job-block__circle">
                   <img
+                    class="job-block__img"
                     src="./src/icons/human-resourse.svg"
                     alt="human resourse"
                   />
+
+                  <img
+                    class="job-block__img_active"
+                    src="./src/icons/human-resourse-active.svg"
+                    alt="human resourse"
+                  />
                 </div>
-                <h2 class="job-block__title">Human Resource</h2>
+                <p class="job-block__title">Human Resource</p>
                 <p class="job-block__subtitle">35 Job Vacancy</p>
               </a>
 
-              <a href="#" class="content-container__job-blocks job-block">
+              <a href="#" class="jobs-group__job-blocks job-block">
                 <div class="job-block__circle">
                   <img
+                    class="job-block__img"
                     src="./src/icons/art&illustration.svg"
                     alt="art&illustration"
                   />
+
+                  <img
+                    class="job-block__img_active"
+                    src="./src/icons/art&illustration-active.svg"
+                    alt="art&illustration"
+                  />
                 </div>
-                <h2 class="job-block__title">Art & Illustration</h2>
+                <p class="job-block__title">Art & Illustration</p>
                 <p class="job-block__subtitle">52 Job Vacancy</p>
               </a>
             </div>
           </div>
 
           <!-- Sign up block -->
-          <div class="second-section__sign-up-block">
+          <div class="categories__sign-up">
```

Comment:
```suggestion
          <div class="sign-up">
```
It's a new separate block which can be used anywhere